### DISCUSSION TITLE
How to make page with url user.github.io

### DISCUSSION THREAD

**Discussion by Wolfyxon** [community/community#23659] [Other Feature Feedback, Questions, & Ideas]
<p>I want <a href="http://user.github.io" rel="noopener nofollow ugc">user.github.io</a> url not a <a href="http://user.github.io/repo" rel="noopener nofollow ugc">user.github.io/repo</a></p>
<p>I see some other pages have this url.<br>
How to do that?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/wolfgames">@WolfGames</a> and welcome to the community.</p>
<p>In order to do this, create a repository in your account named in the following pattern: <code>https://github.com/username/username.github.io</code>. For example, mine is at <a href="https://github.com/lee-dohm/lee-dohm.github.io">https://github.com/lee-dohm/lee-dohm.github.io</a> and for you it would be <a href="https://github.com/WolfGames/wolfgames.github.io">https://github.com/WolfGames/wolfgames.github.io</a>. For more information <a href="https://docs.github.com/free-pro-team@latest/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites">check our GitHub Pages documentation</a>.</p>
<p>Let us know if you have more questions.</p>

**Comment by Wolfyxon** [comment]
<p>Ok thanks so much!<br>
I my webpage finaly has corect url.<br>
Thanks :grinning:</p>
